createEnvironment.js

```diff
@@ -225,6 +225,12 @@ function createEnvironment({ baseEnvironment } = {}) {
 
       const testName = this.getName(obj);
 
+      if (name.includes("failure")) {
+        if (event.error) {
+          this.Sentry.captureException(event.error);
```

Comment:
Will this have the right tags?

Reply:
I updated it so the error is set on the jest test transaction, but yeah, tags should be set correctly (example event https://sentry.io/organizations/sentry/discover/jest:1f61768aea44428d8add34f0eabbf4be/?environment=local&field=title&field=event.type&field=project&field=user.display&field=timestamp&name=All+Events&project=4857230&query=timestamp%3A%3E2022-03-28T13%3A23%3A45%2B00%3A00&sort=-timestamp&statsPeriod=1h&yAxis=count%28%29)